gubernator/gcs_async_test.py

```diff
@@ -53,28 +70,15 @@ def install_handler(stub, structure, base='pr-logs/pull/'):
     def matches(url):
         return url.startswith(gcs_async.STORAGE_API_URL)
 
-    def dispatch(method, url, _payload):
+    def dispatch(method, url, _payload, _headers):
```

Comment:
Fair enough for now, although it looks like this is easy to customize (--dummy-variables-rgx=unused_|_)